### ISSUE TITLE
Restrict possible values for MinutesAuthor

### ISSUE THREAD

**Issue by Bartzi** [fsr-itse/1327#463]
It should be possible to restrict the choices of an author for minutes.
It should only be possible to assign an authors to a minutes document if they have the according edit/add permission

See #459 for the introduction of this feature

**Comment by janno42** [comment]
No, actually we don't want to limit the list of possible numbers that much. For example we need to be able to make a user author of a staff minutes document even if this user is not in the staff group anymore (because the group's members change every year but we might have old documents where we want to set the correct author).

**Comment by Bartzi** [comment]
very good point!
Maybe we could then think about a plain text field for authors, because if somebody left and the database gets cleaned the user object might be deleted, hence the document might not work anymore, or do we want to keep such users forever in this case?

**Comment by janno42** [comment]
we want to keep users forever if they are authors or participants.

**Comment by Bartzi** [comment]
okay, so it should at least not be possible to assign the anonymous user as the author of a minutes document, right?

**Comment by janno42** [comment]
i think so.
